packages/core/usePictureInPicture/index.ts

```diff
@@ -0,0 +1,43 @@
+import { computed, ref } from 'vue-demi'
+
+import type { MaybeElementRef } from '../unrefElement'
+
+import { useEventListener } from '../useEventListener'
+
+export function usePictureInPicture(
+  el: MaybeElementRef<HTMLVideoElement | undefined>,
+) {
+  const video = ref(el)
```

Comment:
Why do we need to add it instead of using `el` with `resolveUnref` directly

Reply:
So we could use unrefElement() ... ?

Reply:
The only issue I have found is the unrefElement doesn't provide reactivity, and as such the `video.requestPictureInPicture()` call doesn't work....